lib/funcmap/funcmap.go

```diff
@@ -42,7 +42,7 @@ import (
 
 // TxtFuncMap returns an aggregated template function map. Currently (custom functions + sprig)
 func SveltosFuncMap() template.FuncMap {
-	funcMap := sprig.FuncMap()
+	funcMap := sprig.TxtFuncMap()
```

Comment:
We cannot change like this as it will a non backward compatible change.

We need to make this optional (by default pick FuncMap). Maybe one way would be to have an annotation that asks for sprig.TxtFuncMap.

For referenced ConfigMap/Secret that would be easy (we have a place where annotation can be set). The problem is for HelmChart (there is no place to add annotation).

Maybe an option would be to have an annotation on the Sveltos resource that requires SveltosFuncMap (ClusterProfile and EventTrigger). So pass an arg to SveltosFuncMap, something like useTxtFuncMap.

Reply:
I have a question regarding the use of goTemplate in sveltos. Specifically, I'm wondering if there are any use cases where generating HTML with goTemplate is intended. Currently, we’re using a FuncMap that includes HTML escaping, and if that behavior is not intentional, perhaps it should be treated as a bug.
I truly value your expertise on this matter and would greatly appreciate any insights or clarifications you could provide. Thank you very much for your time and consideration.